corehq/apps/reports/formdetails/case_metadata.py

```diff
@@ -125,12 +123,13 @@ def add_property_load(self, case_type, name, question_path):
             self.meta.add_property_load(case_type, name, self.form.unique_id, question)
 
 
-class _FormCaseMetadataBuilder(_BaseFormCaseMetadataBuilder):
-    def __init__(self, meta, form, questions):
-        super(_FormCaseMetadataBuilder, self).__init__(meta, form, questions)
+class _RegularFormCaseMetadataBuilder(_BaseFormCaseMetadataBuilder):
```

Comment:
Is there any other code that refers to "regular" forms? I'd rather leave this as `_FormCaseMetadataBuilder` for consistency, or maybe `_BasicFormCaseMetadataBuilder` since "basic" is what we call non-advanced modules.